test/test_container_definition.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+import os
+import time
+import json
+
+from terraform_helper import TerraformHelper
+
+
+REGION = 'eu-west-1'
+
+cwd = os.getcwd()
+
+
+class TestContainerDefinition(unittest.TestCase):
+
+    def setUp(self):
+        self.terraform = TerraformHelper('.')
+        self.terraform.do_get()
+        self.name = 'test-' + str(int(time.time() * 1000))
+        self.vars = {
+            'name': self.name,
+            'image': '123',
+            'cpu': 1024,
+            'memory': 1024,
+            'container_port': 8001
+        }
+
+    def tearDown(self):
+        self.terraform.cleanup()
+
+    def _apply_and_parse_output(self, vars, varsmap):
+        self.terraform.do_apply(vars=vars, varsmap=varsmap)
```

Comment:
The main issue is that this case is more complicated, as I need to pass several variables and create a temporary file to pass the .json with the variables. I will remove the helper, but I guess I will keep a function to call `terraform apply` and parse the output.